robottelo/test.py

```diff
@@ -389,13 +394,40 @@ def _get_vm_list(cls):
         cls.vm_list = vm_list_string.split(',')
 
     @classmethod
-    def _set_testcase_parameters(cls, savepoint_name,
-                                 raw_file_name, stat_file_name):
+    def _set_testcase_parameters(
+            cls,
+            savepoint_name,
+            raw_file_name,
+            stat_file_name,
+            raw_reg=None,
+            stat_reg=None):
+        """Set file names of raw and statistics .csv output
+
+        Each file name is read from `robottelo.constants`
+
+        For subscription by activationKey or deletion case, would output:
+            1. raw data file for ak/del:  raw_file_name.csv
+            2. stat data file for ak/del: stat_file_name.csv
+
+        For subscription by register and attach case, would output:
+            1. raw data file for register:  reg_raw_file_name.csv
+            2. raw data file for attach:    raw_file_name.csv
+            3. stat data file for register: reg_stat_file_name.csv
+            4. stat data file for attach:   stat_file_name.csv
+
+        That is, set extra filenames for register step;
+        while attach step would still output raw/stat_file_name.csv
```

Comment:
Mark inline code with double ticks:

    ``inline code``

Place lists at the same indentation level as other prose.

    """Set file names of raw and statistics .csv output

    Each file name is read from ``robottelo.constants``

    For subscription by activationKey or deletion case, would output:

    1. raw data file for ak/del: raw_file_name.csv
    2. stat data file for ak/del: stat_file_name.csv

    For subscription by register and attach case, would output:

    1. raw data file for register: reg_raw_file_name.csv
    2. raw data file for attach: raw_file_name.csv
    3. stat data file for register: reg_stat_file_name.csv
    4. stat data file for attach: stat_file_name.csv

    That is, set extra filenames for register step;
    while attach step would still output raw/stat_file_name.csv

    """

You can check your work with one of the many reST compilers out there (e.g. http://rst.ninjs.org/). You can learn the basics of reST with [A ReStructuredText Primer](http://docutils.sourceforge.net/docs/user/rst/quickstart.html).